src/env.h

```diff
@@ -263,6 +265,9 @@ class Environment {
     enum Fields {
       // Set this to not zero if the init hook should be called.
       kCallInitHook,
+      // Set this to not zero if async wrap should be enabled. This is
+      // automatically set when SetupHooks is called.
+      kEnabled,
```

Comment:
The name of the field is confusing. `kCallInitHook` is what "enables" `init`/`before`/`after` to be called. This field is an override that forces `before`/`after` to always be called regardless of whether `init` was called, correct?